src/touchid/TouchID.h

```diff
@@ -0,0 +1,60 @@
+#ifndef KEEPASSX_TOUCHID_H
+#define KEEPASSX_TOUCHID_H
+
+#define TOUCHID_UNDEFINED -1
+#define TOUCHID_AVAILABLE 1
+#define TOUCHID_NOT_AVAILABLE 0
+
+#include <QHash>
+#include <QString>
+#include <QByteArray>
+#include <QSharedPointer>
+
+class TouchID
+{
+    public:
+        static TouchID& getInstance()
+        {
+            static TouchID instance; // Guaranteed to be destroyed.
```

Comment:
I much prefer having this function body and global variable declared in TouchID.mm. Declaring it in the header may cause issues if we move this to an external library in the future.